parser.c

```diff
@@ -478,16 +479,43 @@ static struct kanshi_profile *parse_profile(struct kanshi_parser *parser) {
 	}
 }
 
-static struct kanshi_config *_parse_config(struct kanshi_parser *parser) {
-	struct kanshi_config *config = calloc(1, sizeof(*config));
-	wl_list_init(&config->profiles);
+static int parse_config_file(const char *path, struct kanshi_config *config);
+
+static int parse_include_command(struct kanshi_parser *parser, struct kanshi_config *config) {
+	// Skip the 'include' directive.
+	if (!parser_expect_token(parser, KANSHI_TOKEN_STR)) {
+		return -1;
+	}
+
+	if (!parser_read_line(parser)) {
+		return -1;
+	}
 
+	if (parser->tok_str_len <= 0) {
+		return 0;
+	}
+
+	wordexp_t p;
+	if (wordexp(parser->tok_str, &p, 0) == 0) {
```

Comment:
Would be nice to set flags to `WRDE_SHOWERR | WRDE_UNDEF`

Reply:
Note: In local testing I couldn't provoke WRDE_SHOWERR to print any errors, but I added that we at least error on wordexp failure.

Reply:
Ah, that's a good point!